### PR TITLE
Merge nginx+ support into nginx collector and unit tests

### LINKED ISSUE #475: Support for nginx+ status endpoint
The commercial subscription for nginx offers a much more detailed status endpoint and I'd like to get that data into diamond.  I presently have most of the code ready.

The dataset and format is completely different than the normal nginx status info, but the options and testing strategy is pretty similar.  Is there a project preference for...

(a) Adding nginx+ support to the existing nginx collector, either automatically determined or set by a config option and updating tests/documentation

or

(b) Creating an entirely separate nginxplus collector, test cases, and documentation

I can put together a PR for either approach.

### PR DESCRIPTION
**Pull request by travisgroth**
Closes #475

### CODE REVIEW

File: src/collectors/nginx/nginx.py [hunk 1]
```diff
@@ -1,35 +1,65 @@
...
+    </pre>
+
+For commercial nginx+:
+
+    To enable the nginx status page to work with defaults,
+    add a file to /etc/nginx/sites-enabled/ (on Ubuntu) with the
+    following content:
+    <pre>
+    server  {
+        listen *:8080;
```
Comment (shortdudey123) [inline src/collectors/nginx/nginx.py]: bind to `127.0.0.1` like open source example above

**Comment by travisgroth** [comment]
@shortdudey123 are you going to merge this soon or is there some other touching up that needs to be done?

**Comment by shortdudey123** [comment]
haven't had a change to test it myself actually
@jaingaurav you want to take a look at this and give your thoughts on merging?

**Comment by shortdudey123** [comment]
Tested and it looks good to me :)

**Comment by shortdudey123** [comment]
@travisgroth can you rebase to fix the merge conflict?
@josegonzalez can you merge this in after the rebase? i have tested and ran this code on an nginx+ host and it works nicely

**Comment by travisgroth** [comment]
@shortdudey123 @josegonzalez Rebased off master.  I did not add in support for precision since the  'plus' code doesn't calculate any metrics and thus shouldn't wind up with floats.

**Comment by shortdudey123** [comment]
awesome

**Comment by josegonzalez** [comment]
Ouch what a coverage decrease lol.

**Comment by travisgroth** [comment]
@josegonzalez yeah...I'm not clear on how that's possible with this change set.  I'm not very familiar with coveralls and I'm having trouble figuring out what the cause is.

**Comment by shortdudey123** [comment]
@josegonzalez that is an issue w/ all PR's right now, i submitted several last night and the same thing happened on all of them

**Comment by shortdudey123** [comment]
@travisgroth if  you rebase on master, coveralls should pass now
Been running this code for a week and it works great :)

**Pull request merged**